CHANGELOG.md

```diff
@@ -1,9 +1,30 @@
-# dbt_qualtrics version.version
+# dbt_qualtrics v0.4.0
+
+[PR #14](https://github.com/fivetran/dbt_qualtrics/pull/14) and [PR #15](https://github.com/fivetran/dbt_qualtrics/pull/15) includes the following updates:
+
+## Schema & Data Updates
+**2 new columns and 2 schema changes. See [v0.4.0 dbt_qualtrics_source release](https://github.com/fivetran/dbt_qualtrics_source/releases/tag/v0.4.0) release for upstream column addition**
+
+| Data Model | Change Type | Old Name | New Name | Notes |
+| --- | --- | --- | --- | --- |
+| `qualtrics__response` | New column |  | `response_text` | Text of the question response. |
```

Comment:
In the source, the notes say "Captures the free text response associated with the question." I would update here too.

Reply:
Updated.